ui/src/stylesheets/specific/navigation.scss

```diff
@@ -17,15 +17,6 @@
       color: var(--core-violet-4);
     }
   }
-  .nav-list {
-    a.nav-link-active {
-      font-weight: var(--weight-medium);
-      border-left: 2px solid var(--core-violet-4);
```

Comment:
No more border? 😢

Reply:
Imho having the arrows and the border could be too much, would you prefer to keep it?